spec/classes/software_social_skype_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe 'software::social::skype' do
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      let(:facts) do
+        facts
+      end
+
+      context 'with defaults' do
+        if facts[:operatingsystem] == 'Darwin'
+          it { is_expected.to compile.with_all_deps }
+          it { is_expected.to contain_package(/Skype-.*+/).with_provider(appdmg) }
+        elsif facts[:operatingsystem] == 'Ubuntu'
+          it { is_expected.to compile.with_all_deps }
+          it {
+            is_expected.to contain_apt_source('skype-partner')
```

Comment:
There's clearly some syntax issue with `apt_source` here.

@edestecd Can you suggest how to reference `apt::source` with `contains_`?